### PR TITLE
Add long press button to confirm transaction

### PR DESCRIPTION
**Pull request by wojtus7**
Add long press button to confirm transaction. Animation can be customized a little (duration, size of circle and width of stroke but I can add more)

iOS:

https://github.com/daimo-eth/daimo/assets/42337257/4cd911df-4254-442e-bdb5-5b0be50db8cc

Android

https://github.com/daimo-eth/daimo/assets/42337257/c61acbc9-ac34-44b3-bd9c-bc232fffece1

### CODE REVIEW

**Review by dcposch — APPROVED** [review]
Code looks great to me!

cc @michelle-chiu lmk if animation looks right

File: apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  circle: {
+    position: "absolute",
+    transform: [{ rotate: "-90deg" }],
+  },
+  circleBehind: {
+    position: "absolute",
+    transform: [{ rotate: "-90deg" }],
```
[the remark this answers is not in the document]
Comment (wojtus7) [reply apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx:102]: Yup it's always full circle. The reason I use it in both component it that I was afraid that if component is slightly off center for some reason then lack of the rotation would leave circle in different place

[the remark this answers is not in the document]
Comment (wojtus7) [reply apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx]: Checked on both iOS and Android and removal of this line doesn't seem to change anything so removed in current commit 🫡

[the remark this answers is not in the document]
Comment (michelle-chiu) [reply apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx]: @wojtus7 @dcposch Thanks for including me here in the discussion - so I understand correctly, this animation appears when the user holds the "Confirm and Send" button to complete the transfer and also triggers this animation?

On first impression, I'm not a fan of having the user "holding" the button for ~3 secs in order to complete the action (esp if they have to perform multiple transfers at a time). It seems like it adds high friction to this user action and I haven't seen this interaction pattern in other apps before.

If we do move forward with this, I would remove the "Confirm and Send" text when they start holding the button and only show the loading spinner in the design.

[the remark this answers is not in the document]
Comment (dcposch) [reply apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx]: i think it's only~1 second

[the remark this answers is not in the document]
Comment (michelle-chiu) [reply apps/daimo-mobile/src/view/shared/AnimatedCircle.tsx]: In that case, let's change the copy to "Hold and Send" to be more specific about the user action required (as seen in Rainbow's UX).

I'd also like to remove the "Confirm and Send" text when they start holding the button and only show the loading spinner in the design too.

**Pull request merged**